### PR TITLE
Cleanup nephe rbacs

### PR DESCRIPTION
**Pull request by reachjainrahul**
Remove unnecessary/unused role and binding

### CODE REVIEW

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -136,6 +136,16 @@ rules:
...
+- apiGroups:
+    - ""
+  resources:
+    - configmaps
+  verbs:
+    - get
+    - list
+    - watch
+
 - apiGroups:
```
[the remark this answers is not in the document]
Comment (reachjainrahul) [reply config/rbac/role.yaml:149]: yes.. required

**Pull request merged**